src/gui/properties/propertieswidget.cpp

```diff
@@ -156,79 +154,79 @@ PropertiesWidget::PropertiesWidget(QWidget *parent, MainWindow *main_window, Tra
+    m_deleteHotkeyWeb = new QShortcut(QKeySequence::Delete, m_ui->listWebSeeds, 0, 0, Qt::WidgetShortcut);
+    connect(m_deleteHotkeyWeb, SIGNAL(activated()), SLOT(deleteSelectedUrlSeeds()));
+    m_openHotkeyFile = new QShortcut(Qt::Key_Return, m_ui->filesList, 0, 0, Qt::WidgetShortcut);
+    connect(m_openHotkeyFile, SIGNAL(activated()), SLOT(openSelectedFile()));
 }
 
 PropertiesWidget::~PropertiesWidget()
 {
     qDebug() << Q_FUNC_INFO << "ENTER";
-    delete refreshTimer;
-    delete trackerList;
-    delete peersList;
-    delete speedWidget;
-    delete downloaded_pieces;
-    delete pieces_availability;
-    delete PropListModel;
-    delete PropDelegate;
+    delete m_refreshTimer;
+    delete m_trackerList;
+    delete m_peerList;
+    delete m_speedWidget;
+    delete m_downloadedPieces;
+    delete m_piecesAvailability;
+    delete m_propListModel;
+    delete m_propListDelegate;
     delete m_tabBar;
-    delete editHotkeyFile;
-    delete editHotkeyWeb;
-    delete deleteHotkeyWeb;
-    delete openHotkeyFile;
+    delete m_editHotkeyFile;
+    delete m_editHotkeyWeb;
+    delete m_deleteHotkeyWeb;
+    delete m_openHotkeyFile;
     delete m_ui;
     qDebug() << Q_FUNC_INFO << "EXIT";
 }
 
 void PropertiesWidget::showPiecesAvailability(bool show)
 {
-    m_ui->avail_pieces_lbl->setVisible(show);
-    pieces_availability->setVisible(show);
-    m_ui->avail_average_lbl->setVisible(show);
-    if (show || !downloaded_pieces->isVisible())
-        m_ui->line_2->setVisible(show);
+    m_ui->labelPiecesAvailability->setVisible(show);
+    m_piecesAvailability->setVisible(show);
+    m_ui->labelAverageAvailabilityVal->setVisible(show);
+    if (show || !m_downloadedPieces->isVisible())
+        m_ui->lineBelowBars->setVisible(show);
 }
 
 void PropertiesWidget::showPiecesDownloaded(bool show)
 {
-    m_ui->downloaded_pieces_lbl->setVisible(show);
-    downloaded_pieces->setVisible(show);
-    m_ui->progress_lbl->setVisible(show);
-    if (show || !pieces_availability->isVisible())
-        m_ui->line_2->setVisible(show);
+    m_ui->labelDownloadedPieces->setVisible(show);
+    m_downloadedPieces->setVisible(show);
+    m_ui->labelProgressVal->setVisible(show);
+    if (show || !m_piecesAvailability->isVisible())
+        m_ui->lineBelowBars->setVisible(show);
 }
 
 void PropertiesWidget::setVisibility(bool visible)
 {
-    if (!visible && (state == VISIBLE)) {
+    if (!visible && (m_state == VISIBLE)) {
         QSplitter *hSplitter = static_cast<QSplitter *>(parentWidget());
         m_ui->stackedProperties->setVisible(false);
-        slideSizes = hSplitter->sizes();
+        m_slideSizes = hSplitter->sizes();
         hSplitter->handle(1)->setVisible(false);
         hSplitter->handle(1)->setDisabled(true);
         QList<int> sizes = QList<int>() << hSplitter->geometry().height() - 30 << 30;
         hSplitter->setSizes(sizes);
-        state = REDUCED;
+        m_state = REDUCED;
```

Comment:
To get this straight, i should change REDUCED to PropertiesWidget::Reduced?

Reply:
Let's leave it as is until we have rules about it.

Reply:
ok

Reply:
I would replace it with PropertiesWidgetState enum class declared outside the PropertiesWidget class.

Reply:
I better not..